adoc/admin-gpus.adoc

```diff
@@ -181,3 +181,143 @@ Check the kubelet log on the worker node that has a GPU. This may indicate error
 ----
 # journalctl -u kubelet
 ----
+
+== Monitoring
+
+If you have configured link:{docurl}/single-html/caasp-admin/#_monitoring[Monitoring] for your cluster, you may want to use {nvidia}'s Data Center GPU Manager (DCGM) to monitor your GPUs. Follow the steps below to configure the Prometheus exporter and Grafana dashboard for your {nvidia} GPUs.
```

Comment:
Do we want to include a link to https://developer.nvidia.com/dcgm for further information?

The text takes a big step here, and as someone unfamiliar with DCGM I had to look up whether Prom and Graf are requirements for DCGM or optional components or something else.  Maybe a line "The data from DCGM can be exported to the Prometheus instance configured for your cluster." to help give a fuller picture.

Reply:
> Do we want to include a link to https://developer.nvidia.com/dcgm for further information?

Sure

> The text takes a big step here, and as someone unfamiliar with DCGM I had to look up whether Prom and Graf are requirements for DCGM or optional components or something else. Maybe a line "The data from DCGM can be exported to the Prometheus instance configured for your cluster." to help give a fuller picture.

I don't quite follow what's missing here. The line is "If you have configured monitoring..." with a link to how to set up Prometheus and Grafana, "...Follow the steps below to configure the Prometheus exporter and Grafana dashboard". If you have monitoring, you can add this thing to monitor your GPUs. Seems clear to me?

Oh - are you saying that it's not clear that the DCGM *is* the exporter and the dashboard?

Reply:
Added a sentence that hopefully clarifies a bit

Reply:
Yeah, sorry if I wasn't clear (my wordsmithing isn't the best today). I think I was looking for a little bit more detail on the relation between DCGM and Prometheus, and basically I think that is that DCGM includes a metrics exporter which can be configured to work with Prometheus (or apparently some other monitoring systems, but we don't need to mention that).